src/string.cr

```diff
@@ -891,11 +891,30 @@ class String
     end
   end
 
+  # Returns the `Char?` at the given *index*.
```

Comment:
I think we don't need to repeat the return type in the description. It should be plain prose. 
```suggestion
  # Returns the character at *index* or `nil` if it's out of range.
```